### ISSUE TITLE
[Driver 26.19] Accuracy regressions in openai/whisper-tiny and BERT_pytorch

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#3789]
## Summary
Driver 26.19 introduces two new accuracy regressions compared with driver 25.40.

## Affected cases
| Suite | Dtype | Mode | Model | Baseline | Target |
| --- | --- | --- | --- | --- | --- |
| huggingface | bfloat16 | inference | openai/whisper-tiny | pass | fail_accuracy |
| torchbench | amp_bf16 | inference | BERT_pytorch | pass | fail_accuracy |

### Environment setup
```bash
docker run -it -e TZ=Aisa/Shanghai --device=/dev/mem --device=/dev/dri --group-add video --privileged --shm-size=8g intelgpu/ubuntu-24.04-lts2:2523.40 bash

curl -LsSf https://astral.sh/uv/install.sh | env UV_INSTALL_DIR="/usr/local/bin" sh
uv venv myvenv --python 3.12 --clear
source myvenv/bin/activate
uv pip install pip setuptools wheel
uv pip install torch torchaudio torchvision --index-url https://download.pytorch.org/whl/xpu

pytorch_commit="$(python -c 'import torch; print(torch.version.git_version)')"
git clone https://github.com/pytorch/pytorch
cd pytorch
git checkout ${pytorch_commit}

uv pip install pip pandas psutil scipy requests
uv pip install -U numpy==1.26.4 accelerate
uv pip install -r .ci/docker/ci_commit_pins/huggingface-requirements.txt
uv pip install timm

rm -rf torch torch-xpu-ops
git clone https://github.com/intel/torch-xpu-ops
rsync -avz ./torch-xpu-ops/.ci/benchmarks/ ./benchmarks/dynamo/

git clone https://github.com/pytorch/benchmark tmp-benchmark
cd tmp-benchmark
python install.py --continue_on_fail
export PYTHONPATH=$PWD
cd ..

# update torch
# gh --repo intel/torch-xpu-ops run download 26399638675 -p "Torch-XPU-Wheel-*"
uv pip uninstall torch torchaudio torchvision triton-xpu -y
uv pip install Torch-XPU-Wheel-*/*.whl
```

### Accuracy commands
```bash
python benchmarks/dynamo/huggingface.py -d xpu --accuracy --inference --bfloat16 --backend=inductor --disable-cudagraphs --only "openai/whisper-tiny"
python benchmarks/dynamo/torchbench.py -d xpu --accuracy --inference --amp --amp-dtype bfloat16 --backend=inductor --disable-cudagraphs --only BERT_pytorch
```

## Source runs
- Target: https://github.com/intel/torch-xpu-ops/actions/runs/26264076598
- Baseline: https://github.com/intel/torch-xpu-ops/actions/runs/26267132121

## Notes
This issue was created from the Driver 26.19 vs 25.40 comparison report.

**Comment by chuanqi129** [comment]
Driver team is working on reproduce

**Comment by mengfei25** [comment]
Confirmed that used older triton than baseline, so it is not a regression
